README.md

```diff
@@ -6,16 +6,26 @@
 [![npm](https://img.shields.io/npm/v/devextreme-aspnet-data.svg?maxAge=43200)](https://www.npmjs.com/package/devextreme-aspnet-data)
 [![npm nojquery](https://img.shields.io/npm/v/devextreme-aspnet-data-nojquery.svg?maxAge=43200&label=npm+nojquery)](https://www.npmjs.com/package/devextreme-aspnet-data-nojquery)
 
-This extension enables [DevExtreme client-side widgets](https://js.devexpress.com) to consume data from the server in ASP.NET applications. A widget communicates with the server in the following manner: the widget sends data loading options (filtering, grouping, sorting, and other options) to the server, the server processes data according to these options and then sends processed data back to the widget. In this way, you delegate all intensive data operations from the client to the server, thus significantly improving the performance of DevExtreme widgets. This extension can be used directly with the widgets as well as with [ASP.NET MVC Controls](https://js.devexpress.com/Documentation/Guide/ASP.NET_MVC_Controls/).
+This library enables [DevExtreme client-side widgets](https://js.devexpress.com) to consume data from ASP.NET controllers and handlers. A widget communicates with the server in the following manner: 
+- The widget sends data loading options (filtering, grouping, sorting, etc) to the server.
+- The server processes data according to these options and then sends processed data back to the widget.
+
+In this way, you delegate all intensive data operations to the server.
```

Comment:
"In this way"? Which way?

Reply:
Can we rewrite "Thus you delegate all ...." ?

Reply:
You cannot start a sentence with 'Thus'

Reply:
Rewritten: 

> As you can see, all intensive data operations are delegated to the server.

or your variant?

Reply:
@dirk-pieterse Please suggest how we can build a better phrase. We summarize that things described above effectively mean ('in other words') that 'you delegate...'

Reply:
This library enables [DevExtreme client-side widgets](https://js.devexpress.com) to perform CRUD operations via ASP.NET controllers and handlers and allows you to delegate all data-intensive operations to the server:

- the widget sends data loading options (filtering, grouping, sorting, etc) to the server
- the server processes data according to these options
- the processed data is sent back to the widget

OR

This library enables [DevExtreme client-side widgets](https://js.devexpress.com) to perform CRUD operations via ASP.NET controllers and handlers and allows you to delegate all data-intensive operations to the server. To do this,  the widget sends data loading options (filtering, grouping, sorting, etc) to the server, the server processes data according to these options, and the processed data is sent back to the widget.